Bottom: 7ae1fbe3cf953f2bc4fe34ca7036ba66bb9ec43a
Top:    14b527b167cd8f59e77865a591f646ec1c3c1c48
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-23 17:06:09 +0800

Refresh of autofs-5.0.7-fix-portmap-lookup.patch

---

diff --git a/include/rpc_subs.h b/include/rpc_subs.h
index 1d5357c..b6d59f9 100644
--- a/include/rpc_subs.h
+++ b/include/rpc_subs.h
@@ -39,8 +39,6 @@
 #define RPC_CLOSE_ACTIVE	RPC_CLOSE_DEFAULT
 #define RPC_CLOSE_NOLINGER	0x0001
 
-extern const rpcvers_t rpcb_version;
-
 #define PMAP_TOUT_UDP	3
 #define PMAP_TOUT_TCP	5
